contents/blog/the-state-of-plugins.md

```diff
@@ -0,0 +1,81 @@
+---
+date: 2021-11-28
+title: The state of plugins on PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+Author: yakko-majuri
```

Comment:
can we get shared credits? 

As someone who is a writer [[1]](https://yakkomajuri.medium.com/) [[2]](https://yakkomajuri.github.io/), I'm very uncomfortable taking full credit for something I didn't fully create :D

Reply:
[Actually, no.](https://github.com/PostHog/posthog.com/issues/2402) 

I'm totally happy not to be credited. You did all the heavy lifting on the first version and I'm very used to ghostwriting anyway.

Reply:
I should add: I'm very happy to take feedback on how you would write it, or for you to add more of your personal style in. If you do feel it's far from your personal style then we can credit it to me - but I'll then add an opening which makes it clear that this is based on work you originally did.